### DISCUSSION TITLE
Delete a configured tier

### DISCUSSION THREAD

**Discussion by Feiniu6ABC** [minio/minio#14025] [Q&A] [answered]
Hi, I cannot find the delete button for deleting a configured tier, is there an API for doing this?

**Comment by klauspost** [comment, marked as answer]
That would make tiered objects unavailable. You can edit existing tiers, but not remove them.

**Reply by Feiniu6ABC** [reply]
Thanks for your reply, but it doesn't make sense that even if the tier endpoint is no longer used, the configured tiers still cannot be removed.  This web page will have many many unused tiers if they use MinIO for a long time
![企业微信截图_16414343875345](https://user-images.githubusercontent.com/41153766/148316021-ffe1b776-679c-448b-bb04-cccc262186ea.png)

**Reply by harshavardhana** [reply]
> Thanks for your reply, but it doesn't make sense that even if the tier endpoint is no longer used, the configured tiers still cannot be removed. This web page will have many many unused tiers if they use MinIO for a long time

No, we do not allow it @Feiniu6ABC that's part of the design.  When using Tiering make sure that you know what you are doing.

**Reply by ravindk89** [reply]
@Feiniu6ABC we cannot know that a tier is completely unused. Providing a delete button, even with many safety checks "are you sure, please make sure, etc." will inevitably result in someone accidentally removing an active tier and losing data. We also expect typical tiering usage to consist of a few tiers that remain constant over the lifespan of the MinIO deployment. Based on that, the cost-benefit of a "delete" button is low compared to the risk of data loss due to simple human errors - which occur more frequently than one might expect.

We appreciate your understanding here.
